netci.groovy

```diff
@@ -122,9 +122,11 @@ def CreateXPlatBuildTask = { isPR, buildType, staticBuild, machine, platform, co
     def infoScript = "bash jenkins/get_system_info.sh --${platform}"
     def buildFlag = buildType == "release" ? "" : (buildType == "debug" ? "--debug" : "--test-build")
     def staticFlag = staticBuild ? "--static" : ""
+    def swbCheckFlag = (platform == "linux" && buildType == "debug" && !staticBuild) ? "--wb-check" : "";
```

Comment:
More wanted to avoid the trouble configuring CI machine and fixing the plugin cmake file to support multiple platforms (I once got it to work on OSX, but configuration was more difficult and not sure if it still works). If there is any issue detectable by this plugin, one variant should work.